assets/_scss/explainer.scss

```diff
@@ -138,3 +153,58 @@
         padding-top: 5%;
     }
 }
+
+/* TL;DR box */
+
+.longread .tldr {
+    background-color: $c-extralight-blue;
+    border-radius: 1rem;
+    margin-block: 2rem;
+    padding: 1.5rem;
+    &, li {
+        line-height: 1.35;
+    }
+    .title {
```

Comment:
I'm not thrilled about the look of the title. It looks a bit pale at the moment. I propose something in the following direction (`font-weight: bold; letter-spacing: .05em; text-transform: uppercase;`). What do you think?
![image](https://user-images.githubusercontent.com/134321/208667037-d7db7fdd-9afe-4294-b6d8-50b3b784d174.png)

Reply:
Does it mean also removing the `opacity` param? I don't feel strongly about it so I've committed this change.

Reply:
Yes, the opacity made it a bit too faint in my opinion.